src/MeetingsSDKAdapter.js

```diff
@@ -661,6 +671,10 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
               }),
             }));
         } else {
+          // update the meeting title if available after password verification
```

Comment:
Why did we need this code and above now?

Reply:
removed from above